.gitignore

```diff
@@ -34,3 +34,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+.tool-versions
+.idea
```

Comment:
Which gems generate this output?

Reply:
Thanks for the review! .idea is generated by my IDE (rubymine) and tool-versions by my version manager (asdf)